project/Component/NefCarbon/App/CarbonSyncDownloader.swift

```diff
@@ -32,6 +32,10 @@ class CarbonSyncDownloader: CarbonDownloader, CarbonViewDelegate {
         return syncResult
     }
     
+    func carbon(configuration: CarbonModel) -> IO<CarbonError, Image> {
+        fatalError()
```

Comment:
I guess this will be added in a future PR, right? I suggest adding a `#warning` with a TO-DO message to help us not to forget this needs to be addressed.

Reply:
Great! I love that idea, yes, it will be done in the `NefCarbo` module refactor.